modules/m_stats.c

```diff
@@ -1523,12 +1523,14 @@ stats_ltrace(struct Client *source_p, int parc, const char *parv[])
 	const char *name;
 	char statchar = parv[1][0];
 
+	if (ConfigFileEntry.stats_l_oper_only == 2 && !IsOperGeneral(source_p))
+		sendto_one_numeric(source_p, ERR_NOPRIVILEGES, form_str(ERR_NOPRIVILEGES));
```

Comment:
Without a `return` this privilege check doesn't appear to enforce anything.

```suggestion
	if (ConfigFileEntry.stats_l_oper_only == 2 && !IsOperGeneral(source_p)) {
		sendto_one_numeric(source_p, ERR_NOPRIVILEGES, form_str(ERR_NOPRIVILEGES));
		return;
	}
```

Reply:
oops :)

Reply:
alternatively the change might be a following `else if`